prow/plugins/trigger/trigger.go

```diff
@@ -50,8 +50,11 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 		if len(parts) != 2 {
 			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
 		}
-		trusted, _ := trustedOrgForRepo(config, parts[0], parts[1])
-		configInfo[repo] = fmt.Sprintf("The trusted Github organization for this repository is %q.", trusted)
+		org, repo := parts[0], parts[1]
+		if trigger := config.TriggerFor(org, repo); trigger != nil && trigger.TrustedOrg != "" {
+			org = trigger.TrustedOrg
+		}
+		configInfo[repo] = fmt.Sprintf("The trusted Github organization for this repository is %q.", org)
```

Comment:
You can't use `repo` as the map key anymore. You need the full `org/repo` name, but you redefine `repo` to be just the repo name.

Reply:
Done